deployment/ansible/group_vars/all

```diff
@@ -3,7 +3,7 @@ postgresql_support_libpq_version: "9.4.0-1.pgdg14.04+1"
 app_username: "vagrant"
 
 # open trip planner vars
-otp_process_mem: 4G
+otp_process_mem: 2G
```

Comment:
Potentially useful, although you may not care to do it here – Ansible has a [`lookup`](http://docs.ansible.com/playbooks_lookups.html#more-lookups) function that can pull from the environment where Ansible is being executed.